pkg/kv/bulk/sst_batcher.go

```diff
@@ -227,11 +227,10 @@ func (b *SSTBatcher) AddMVCCKey(ctx context.Context, key storage.MVCCKey, value
 		return err
 	}
 
-	// If we do not allowing shadowing of keys when ingesting an SST via
-	// AddSSTable, then we can update the MVCCStats on the fly because we are
-	// guaranteed to ingest unique keys. This saves us an extra iteration in
-	// AddSSTable which has been identified as a significant performance
-	// regression for IMPORT.
+	// If we do not allow shadowing of keys when ingesting an SST via AddSSTable,
+	// then we can update the MVCCStats on the fly because we are guaranteed to
+	// ingest unique keys. This saves us an extra iteration in AddSSTable which
+	// has been identified as a significant performance regression for IMPORT.
 	if !b.disallowShadowingBelow.IsEmpty() {
 		b.updateMVCCStats(key, value)
 	}
```

Comment:
For posterity, even though we don't update stats here, they're computed by the client when flushing:

https://github.com/cockroachdb/cockroach/blob/e2c163b2fe74eede6e2a75f015dde751d7cd05ad/pkg/kv/bulk/sst_batcher.go#L613-L618

If `nil` is passed to the server, it would have computed them, but it's unclear if it would have when passing empty stats (something would have to replace the empty request parameter with `nil` somewhere). But that doesn't matter -- the SST appears like it should be sent with MVCC stats included.